ui/packages/tidb-dashboard-lib/src/apps/Metrics/data/MetricsItems.tsx

```diff
@@ -1,9 +1,9 @@
 const MetricsItems = [
```

Comment:
For non-component, we prefer to use camel naming style for variable and file, for example: `metricsItems`